### DISCUSSION TITLE
Queries related to Pull Requests

### DISCUSSION THREAD

**Discussion by aryashah2k** [PatternHouse/CPlusPlus-PatternHouse#682] [Q&A] [answered]
Add queries realted to PRs, issues and any other contribution guidelines doubts

**Comment by anshyyy** [comment]
for issue #353 I'm not getting the option to compare and pull as like before.

**Reply by aryashah2k** [reply]
could you elaborate more as in what exactly is happening? What can you see as of now on your branch of the repo?

**Reply by aryashah2k** [reply]
If your repository is few commits behind, then pull origin, so that all recent commits are updated.
Then try again, if it does not work then you could try creating a new branch or fork the repo again in extreme case

**Comment by aryashah2k** [comment, marked as answer]
I hope you found the problem to the solution and everything is working proper! @anshyyy

**Reply by anshyyy** [reply]
yeah thanks... also I want to ask if I commit two issues consecutively without created a PR. Now if I create a pull request then both commits will get to it's corresponding issues? or something else??

**Reply by aryashah2k** [reply]
If you commit code files for more than one pattern in a single PR, you can mention the issue numbers in your PR and I will take care of that

**Comment by psahithireddy** [comment]
Hi @aryashah2k can you please add back readmes for C++ patterns, even though issues are incorrectly linked, all the patterns are in order, unlike on the website, which makes it difficult to view patterns while writing codes.
Moreover, how we assign issues to ourselves? By opening a pull request or self assigning at issues (it's not showing "assign yourself")?

**Reply by aryashah2k** [reply]
Hi, so what you can do is to view the patterns, refer the python patternhouse readmes, all patterns are same in sequence and you will not have to visit the website which is still under massive development 
and the way you get to assign yourself an issue is tag me in that particular issue by searching them in the issues tab

**Comment by aryashah2k** [comment]
I know it is tedious but that's the only way I can save some efforts for you because those readmes oin c++ created massive confusion for other participants and things got difficult to clarify for them all!
